spec/views/holds/index.html.erb_spec.rb

```diff
@@ -7,56 +7,12 @@
 
   before do
     allow(hold).to receive(:item_type_mapping).and_return(ITEM_TYPE_MAPPING)
+    assign 'patron_key', '123'
   end
 
-  context 'when there are holds ready to pickup' do
-    before do
-      assign(:holds_ready, [hold])
-    end
+  it 'has a creates a div with a data attribute containing the patron_key' do
+    render
 
-    it 'displays a hold' do
```

Comment:
instead of these feature specs for holds display?

Reply:
yes because they require javascript

Reply:
yeah sure, I did not see any new added so I was meaning to ask will you add any? Or do you think the ones we already have there is already covering display holds as well?

Reply:
right that's what I was thinking, all of the existing specs would fail if this view holds didn't work, it's getting tested implicitly by actions like 'clicking a checkbox'